rfcs/044-sierra-series/README.md

```diff
@@ -0,0 +1,503 @@
+     {   // 830 & 773 contain the same information
+        "title": "ACLS Humanities E-Book.",
+        "type": "Work" //or Series?
+      }
+    ]
+    ```
+- [2125597](https://search.wellcomelibrary.org/iii/encore/record/C__Rb2125597?lang=eng&suite=cobalt&marcData=Y)
+  490 with ISSN + 830 and 773. There are 1037 bibs that have the same ISSN in a series statement but
+  with different volume subfield.
+```yaml
+"partOf": [
+  {   // 490 +830 contain duplicated info
+    "title": "Methods in Molecular Biology, Methods and Protocols; 212",
+    "partOf": [
+      {
+        "id": "abcdefg",
+        "identifiers": [
+          {
+            "identifierType": {
+              "id": "issn",
+              "label": "ISSN",
+              "type": "IdentifierType"
+            },
+            "value": "1064-3745",
+            "type": "Identifier"
+          }
+        ],
+        "title": "Methods in Molecular Biology, Methods and Protocols",
+        "type": "Series"
+      }
+    ],
+    "type": "Series"
+  },
+  {   // 773
+    "title": "Springer eBooks",
+    "type": "Work"
+  }
+]
+```
+- [3001878](https://search.wellcomelibrary.org/iii/encore/record/C__Rb3001878?lang=eng&suite=cobalt&marcData=Y)
+  Only 830 with no 490/440 or 773
+```yaml
+"partOf": [
+    {
+        "title": "Adamson Collection Wellcome Library", // subfield $a concatenated with $p
+        "type": "Series"
+    }
+]
+```
+- [1110225](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1110225?lang=eng&suite=cobalt&marcData=Y)
+Series statement in 440 with only the title
+```yaml
+"partOf": [
+    {
+        "title": "Monographs on inorganic and physical chemistry",
+        "type": "Series"
+    }
+]
+```
+- [1204561](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1204561?lang=eng&suite=cobalt&marcData=Y) 440 with id
+```yaml
+"partOf": [
+  {
+    "title": "Brill's studies in intellectual history; v. 13",
+    "partOf": [
+      {
+        "id": "abcdefgh",
+        "identifiers": [
+          {
+            "identifierType": {
+              "id": "issn",
+              "label": "ISSN",
+              "type": "IdentifierType"
+            },
+            "value": "0920-8607",
+            "type": "Identifier"
+          }
+        ],
+        "title": "Brill's studies in intellectual history",
+        "type": "Series"
+      }
+    ],
+    "type": "Series"
+  }
+]
+```
+- [2301867](https://search.wellcomelibrary.org/iii/encore/record/C__Rb2301867?lang=eng&suite=cobalt&marcData=Y)
+  773 not overlapping with series and no id.
+```yaml
+"partOf": [
+    {
+        "title": "Eighteenth Century collections online",
+        "type": "Work"
+    }
+]
+```
+- [1186777](https://search.wellcomelibrary.org/iii/encore/record/C__Rb1186777?lang=eng&suite=cobalt&marcData=Y)
+  773 with an id in subfield $w which links to another Work in our library (see example below)
+```yaml
+"partOf": [
+   {
+    "title": "Basil Hood. Photograph album; page 9",
+    "partOf": [
```

Comment:
I'm not sure the nested partOf should be a list.  Particularly considering that the title above seems to refer to the part in context of what it is part of.

e.g. 
A portrait of Basil Hood in a white coat outside Charing Cross Hospital might belong to any number of containers (portraits of BH, photos of CXH ...), so a partOf list is needed at the top level.

I would expect that Page 9 of the Basil Hood Photo Album is only ever going to be part of the Basil Hood Photo Album. (not Top 10 ninth pages).

This actually calls into question the need to have nested partOf objects. If we were to rename the part title, then all of the properties in the nested partOf could be promoted to the top-level partOf.

Reply:
Looking wider, at the idea that this construct also serves to put a journal article into a particular volume of a journal, without having to maintain a record for that specific volume:

What would happen if that volume becomes significant in its own right and deserves a record?  Is that a thing that might happen (collection: journal volumes with discredited articles)?